src/styles/components/service-table/styles.less

```diff
@@ -44,7 +44,7 @@
     }
 
     .service-status-icon-wrapper {
-      width: 110px;
+      min-width: 110px;
```

Comment:
In the initial version of this PR, the status was "Deploying (Delayed)". The long string was causing a visual bug with the progress bar. I am changing it to min-width to avoid such problems in the future.
![Screenshot from 2019-05-27 11-15-29](https://user-images.githubusercontent.com/40791275/58416815-84e63680-808b-11e9-938e-c55fe16b60ca.png)